src/PaletteCell.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2023 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+import { html } from "htm/preact";
+import "./PaletteCell.scss";
+
+export function PaletteCell (props) {
```

Comment:
Regarding the component name, I was thinking it uses the same name as the cell type it supports. The name starts with a more general category performed by the cell. For example, keys that perform actions when pressed are in "action" category, so this cell type and component for BMW code keys are named "ActionBmwCodeKeys"; names for command buttons start with "Command..."; names for display areas start with "Content...". There might be better names than "action", "command" and "content". This naming convention will make files in the `/src` folder easy to navigate since all components in the same category will stay together. What do you think?

Reply:
I like the idea, especially the way that files are grouped within the `/src` folder.  "Command" and "Content" are fine, but "Action" is too generic, I think.  I'm leaning towards "Insert" since the action is to insert content into encoding area and, ultimately, into the message area.

Still, there is nothing within "Insert" to communicate the idea of entering a _sequence_ of codes.  Then again, I'm not sure there needs to be.